HowTos/aml-compute-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# This script installs msticpy in AML Compute
+LOG=/tmp/aml-setup-msticpy.log
+echo ======================================== 2>&1 | tee -a $LOG
+echo Started $(date) 2>&1 | tee -a $LOG
+echo Installing libs 2>&1 | tee -a $LOG
+sudo apt-get --yes -q install libgirepository1.0-dev 2>&1 | tee -a $LOG
+sudo apt-get --yes -q install gir1.2-secret-1 2>&1 | tee -a $LOG
+
+ENVIRONMENT=azureml_py38
+sudo -u azureuser -i <<EOF
+LOG=/tmp/aml-setup-msticpy.log
+ENVIRONMENT=azureml_py38 
+echo ---------------------------------------- 2>&1 | tee -a $LOG
+conda activate "$ENVIRONMENT"
+echo ---------------------------------------- 2>&1 | tee -a $LOG
+echo Updating $ENVIRONMENT 2>&1 | tee -a $LOG
+conda install --yes --quiet pip wheel 2>&1 | tee -a $LOG
+echo Removing enum34 and installing pygobject 2>&1 | tee -a $LOG
+
+pip uninstall --yes enum34 2>&1 | tee -a $LOG
+conda install --yes --quiet -c conda-forge pygobject 2>&1 | tee -a $LOG
+
+echo Installing msticpy in $ENVIRONMENT 2>&1 | tee -a $LOG
+pip install msticpy 2>&1 | tee -a $LOG
```

Comment:
Given time isn't as much of an issue here do we want to install some extras at this point. Given the majority of users will be coming from Sentinel and using Azure I could see the the 'azuresentinel' being useful. We do this below for 3.6 so seems to make sense to also do it for 3.8

Reply:
Good spot. I think that was just a slip of the pen. I'm sure I intended to do it for both.